src/fs.cpp

```diff
@@ -25,11 +25,18 @@
 #include <mapnik/util/fs.hpp>
 
 #include <mapnik/warning.hpp>
+
+#if __cplusplus >= 201703L && !defined(USE_BOOST_FILESYSTEM)
+#include <filesystem>
+namespace fs = std::filesystem;
+#else
 MAPNIK_DISABLE_WARNING_PUSH
 #include <mapnik/warning_ignore.hpp>
 #include <boost/filesystem/operations.hpp> // for absolute, exists, etc
 #include <boost/filesystem/path.hpp>       // for path, operator/
 MAPNIK_DISABLE_WARNING_POP
+namespace fs = boost::filesystem;
+#endif
```

Comment:
maybe we should add a file in `include/mapnik/fs.hpp` with 
```cpp
#if defined(__cpp_lib_filesystem) && !defined(USE_BOOST_FILESYSTEM)
#include <filesystem>
#else
#include <boost/filesystem/operations.hpp> // for absolute, exists, etc
#include <boost/filesystem/path.hpp>       // for path, operator/
#endif

namespace mapnik {
#if defined(__cpp_lib_filesystem) && !defined(USE_BOOST_FILESYSTEM)
namespace fs = std::filesystem;
#else
namespace fs = boost::filesystem;
#endif
}
```
and then just include it everywhere where we need the filesystem?

Reply:
https://github.com/mapnik/mapnik/pull/4383/commits/a244effa91c04f9e46e506e72213fbd1080516af